app/assets/stylesheets/fae/modules/forms/_label.scss

```diff
@@ -18,4 +18,19 @@ label {
     font-style: italic;
     line-height: 1.3;
   }
+
+  &.label--two_col {
+    display: flex;
+
+    .label_inner {
+      flex-shrink: 0;
+      max-width: 50%;
```

Comment:
`flex: 0 0 50%` isn't resulting in the desired behavior, lending too much space to the helper text vs. the label. Unless this is a breaker, going to leave `flex-shrink` and `max-width` in place